common/src/main/java/com/wynntils/core/features/overlays/OverlayManager.java

```diff
@@ -54,13 +61,57 @@ private static void renderOverlays(RenderEvent event, OverlayInfo.RenderState re
         for (Overlay overlay : enabledOverlays) {
             OverlayInfo annotation = overlayInfoMap.get(overlay);
 
-            if (renderState != annotation.renderAt() || event.getType() != annotation.renderType()) continue;
+            if (annotation.renderAt() == OverlayInfo.RenderState.Replace) {
```

Comment:
You lost the `event.getType() != annotation.renderType()` in the process :(

Reply:
Ooops. Fixed.